client/src/i18n/en/form.json

```diff
@@ -293,7 +295,8 @@
             "CONFIRMATION": "Confirmation",
             "CONTACT": "Contact",
             "CONTACT_ADMIN": "Please contact the administrator",
-            "COST": "Cost",
+            "CONVERTION_RATE": "Convertion rate",
```

Comment:
@mbayopanda  Try to update my branch, I modified the system, it can only be configured with the currency of the company

![image](https://user-images.githubusercontent.com/8035588/49934039-98cb2100-fecd-11e8-92ba-5625b255e2c0.png)

Reply:
@mbayopanda The conversion rate is set only if you have selected a currency other than the company currency in the configuration interface 
![image](https://user-images.githubusercontent.com/8035588/49934501-d4b2b600-fece-11e8-8d31-d4284fe0549d.png)